Sources/SwiftlyCore/Platform.swift

```diff
@@ -88,7 +88,12 @@ extension Platform {
 
     /// The "toolchains" subdirectory of swiftly's home directory. Contains the Swift toolchains managed by swiftly.
     public var swiftlyToolchainsDir: URL {
+        #if !os(macOS)
```

Comment:
Can we invert this?

```swift
#if os(macOS)

#elseif os(Linux)

#else
  #error("Unsupported platform")
#endif
```

I find that generally cognitively easier to parse, but also assume we'll be adding Windows at some point.

Reply:
This has been replaced with a Platform abstraction for both the toolchain and bin directories.